sources/packages/backend/apps/api/src/route-controllers/designation-agreement/models/designation-agreement.dto.ts

```diff
@@ -72,9 +78,10 @@ export class SubmitDesignationAgreementAPIInDTO {
  * be validated by the form.io dryrun validation.
  */
 export class SubmittedLocationsAPIInDTO {
-  @Allow()
+  @IsNumber()
```

Comment:
My understanding is that `@IsNumber()` is already validated by the `@IsPositive()`.
The other places along the API using `@IsPositive()` are also not used in combination with the `@IsNumber()`.